.travis.yml

```diff
@@ -1,13 +1,13 @@
 sudo: required
-dist: trusty
+dist: xenial
```

Comment:
[GCC 5.1 changed its C++ ABI](http://allanmcrae.com/2015/06/the-case-of-gcc-5-1-and-the-two-c-abis/) which resulted in [problems between newer packages and older versions of Ubuntu which still used the old ABI](https://stackoverflow.com/questions/36816570/glibcxx-3-4-21-not-defined-in-file-libstdc-so-6-with-link-time-reference). To avoid these issues I updated the Ubuntu distro that we use during testing to Xenial, and updated the distro-specific packages below.